sdk/guestconfiguration/Azure.ResourceManager.GuestConfiguration/src/autorest.md

```diff
@@ -54,4 +59,10 @@ rename-rules:
   URI: Uri
   Etag: ETag|etag
 
+directive:
+  - remove-operation: GuestConfigurationAssignmentsVMSS_List
```

Comment:
Should create an issue of why the list method not work properly here

Reply:
Created: https://github.com/Azure/autorest.csharp/issues/2560